tools/ci/ci-nightly.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for license information.
+
+# MixedReality-SpectatorView build native content
+
+# Scheduled run, always run to monitor churn in external dependencies
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily Midnight Build
+  branches:
+    include:
+    - master
+  always: true
+
+# Trigger CI on push changes
```

Comment:
Nit: It feels like this comment and the trigger of none don't seem to match up - i.e. it seems like this code/comment do match up:

https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml#opting-out-of-ci-builds